src/modules/Builder/features/AboutApplet/AboutApplet.test.tsx

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+
+import { renderWithAppletFormData } from 'shared/utils/renderWithAppletFormData';
+import { page } from 'resources';
+import { mockedAppletFormData, mockedAppletId } from 'shared/mock';
+
+import { AboutApplet } from './AboutApplet';
+
+const route = `/builder/${mockedAppletId}/about`;
+const routePath = page.builderAppletAbout;
+
+describe('AboutApplet', () => {
+  test('should validate applet name', async () => {
+    renderWithAppletFormData({
+      children: <AboutApplet />,
+      appletFormData: mockedAppletFormData,
+      options: { route, routePath },
+    });
+
+    const appletName = screen.getByLabelText('Applet Name');
+    fireEvent.change(appletName, { target: { value: '' } });
+
+    await waitFor(() => expect(screen.getByText('Applet Name is required')).toBeInTheDocument());
+  });
+});
```

Comment:
could you please add more tests? e.g. that all fields are rendered (for new applet/for editing applet); all changes are applied to the appropriate properties in form data; theme